src/components/vault/EnhancedChatBubble.tsx

```
@@ -0,0 +1,148 @@
+  // Get badge style based on sender type
+  const getBadgeStyle = () => {
+    if (isAgent) {
+      return "bg-green-100 text-green-600 dark:bg-green-800/50 dark:text-green-300";
+    } else if (isAdmin) {
+      return "bg-blue-100 text-blue-600 dark:bg-blue-800/50 dark:text-blue-300";
+    } else {
+      return "bg-gray-100 text-gray-600 dark:bg-gray-800/50 dark:text-gray-300";
+    }
+  };
+
+  const renderReasoningLog = (log: LogEntry) => {
```

ESLint

[error] 'renderReasoningLog' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)